cmd/re/main.go

```diff
@@ -50,6 +51,11 @@ const (
 	defSvcHTTPPort = "9008"
 )
 
+var (
+	alarmsPrefix  = "a"
+	writersPrefix = "w"
```

Comment:
We can use full words here `alarms` and `writers`. We are using short versions in messaging API to preserve some bandwidth and battery for devices.